src/constants/FeatureFlagEnum.ts

```diff
@@ -1,15 +1,8 @@
 import { getConfig } from 'config'
 
-export enum FlagValue {
-  On = 'on',
-  Off = 'off'
-}
-
 export type Flag = typeof FeatureFlagEnum[keyof typeof FeatureFlagEnum]
 
 export const FeatureFlagEnum = {
```

Comment:
while you're at it, do you wanna rename this to just `FeatureFlag`?

Reply:
Was going to comment the exact same :D digged in the codebase and except for a few cases, we seem to mostly use the naming without `Enum` prefix.

Reply:
yeah sure, both the flag and enum?

Reply:
yes please